keyboards/planck/keymaps/fabian/keymap.c

```diff
@@ -14,13 +14,13 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#pragma message "You may need to add LAYOUT_planck_grid to your keymap layers - see default for an example"
 #include "planck.h"
```

Comment:
Could you replace these includes with just `#include QMK_KEYBOARD_H`, and the include for the muse.h file ?